dex/tests/test.bats

```diff
@@ -11,6 +11,10 @@ load helpers
 }
 
 @test "OpenID Connect discovery document is returned with issuer set" {
-  curl http://127.0.0.1:5556/dex/.well-known/openid-configuration |
-    jq -e '.issuer | test("https://localhost/dex")'
+  jq -ne --argjson doc "$(curl http://127.0.0.1:5556/dex/.well-known/openid-configuration)" \
+    '$doc.issuer | test("https://localhost/dex")'
```

Comment:
What was here _before_ would have **not** caught an error where `curl` had _no output_. Using `--argjson doc "$(curl ...)"`, this will be caught.